chain-tx-enclave-next/tx-validation-next/src/sgx_module.rs

```diff
@@ -19,6 +19,7 @@ use std::sync::Arc;
 
 /// FIXME: genesis app hash etc.?
 pub const NETWORK_HEX_ID: u8 = get_network_id!();
+pub const TDBE_MRENCLAVE: &[u8; 32] = include_bytes!("tdbe.mrenclave");
```

Comment:
`get_tdbe_mrenclave` proc macro can be removed (in the enclave-macro)
and this call can be replaced with `TDBE_MRENCLAVE`
https://github.com/crypto-com/chain/blob/master/chain-tx-enclave-next/tx-validation-next/src/sgx_module.rs#L38

Reply:
done